leaf-ui/rappid-extensions/BackboneModelsOther.js

```diff
@@ -31,7 +31,8 @@
     },
 
     defaults: {
-        linkType: 'and',
+        displayType: "element",
```

Comment:
Is there a difference between using `""` and `''` here? If not could change so it is consistent with the other quotation marks

Reply:
Made consistent.